method/application/accept.js

```diff
@@ -0,0 +1,34 @@
+const applicationSpec = require('../../spec/application/application')
+
+module.exports = function AcceptGroupApplication (server) {
+  return function acceptGroupApplication (applicationId, text, cb) {
+    server.tribes.application.get(applicationId, (getErr, applicationData) => {
+      const { groupId, recps } = applicationData
+      server.tribes.invite(groupId, recps, { text }, (err, invite) => {
```

Comment:
I think this is a mistake - I need to check what's in  `applicationData` but we only want the recps of the person applying to be invited.

while in the `content` below we want the recps to be : the person applying + the kaitiaki who this was sent to

Reply:
from reading code looks like you want to use `[ applicationData.applicantId ]` instead of `recps`

Reply:
That's something I meant to ask u. Does the person who sends a private message need to put itself as a `recps`? Judging from this I guess not.

Reply:
Got my answer from the comment on create. But since the returned `recps` will already have the applicantId there's not need to add it again here.

Reply:
Technically all the authors are already in the group, and this message will be encrypted to `[group, ...newPeople]`. So what you've written works, but you would be more accurate to just pass in `[ applicationData.applicantId ]` I think.

As this is not urgent I'll leave this and maybe tweak it later